### ISSUE TITLE
Switch locations based on time

### ISSUE THREAD

**Issue by ironjan** [ironjan/MensaUPB#17]
The location and day shown after start could switch based upon time ( today location -14, Abendmensa from 14-20,  tomorrow location 20+).

Partly depends on #16.

**Comment by ironjan** [comment]
Won't fix for now. Was not yet requested by users (except for me).
